dashboard/app/views/pd/application/teacher_application_mailer/principal_approval_teacher_reminder.html.haml

```diff
@@ -17,13 +17,13 @@
 %p
   As a reminder, for teachers participating in the program, we require the following commitment:
   %ul
-    %li Attend one, five-day summer workshop in 2019
-    %li Attend up to four, one-day local workshops during the 2019-20 school year (usually held on Saturdays)
-    %li You hope to teach the course in the 2019-20 school year
+    %li Attend one, five-day summer workshop
+    %li Attend up to four, one-day local workshops during the school year (usually held on Saturdays)
+    %li To teach the course in the school year
```

Comment:
NIT: This sentence ended up kinda funky. Maybe just "Teach the course..." would make more sense?

Reply:
good call